local/config/finna/WayfinderService.ini.sample

```diff
@@ -2,20 +2,23 @@
 
 [General]
 enabled = false
-adapter = "WayfinderSampleAdapter"
 
 [sample]
 wf_url = "http://wf-hambib.stg.easyscreen.io/includes"
 marker_url = "http://wf-hambib.stg.easyscreen.io"
+adapter = "WayfinderSampleAdapter"
 
 [piki]
 wf_url = "http://wf-piki.stg.easyscreen.io/includes"
 marker_url = "http://wf-piki.stg.easyscreen.io"
+adapter = "WayfinderPikiAdapter"
```

Comment:
I'd expect most libraries that use Axiell Aurora to follow the same pattern. Do you think it would make sense to call the adapter something like `WayfinderAuroraAdapter`? Otherwise I'm afraid we'll end up in a situation where a dozen libraries use "Piki" adapter.